backend/routes/routes.go

```diff
@@ -1,11 +1,17 @@
 package routes
 
-func InitRoutes() {
-	InitIndexerRoutes()
-	InitCanvasRoutes()
-	InitPixelRoutes()
-	InitWebsocketRoutes()
-	InitTemplateRoutes()
-	InitUserRoutes()
-	InitContractRoutes()
+import (
+  "github.com/gorilla/mux"
```

Comment:
I'm curious about switching to `gorilla/mux.` Does it help a lot with your issue, or is it just a change you think will be useful in the future?